pkg/metrics/metrics.go

```diff
@@ -0,0 +1,131 @@
+package metrics
+
+import (
+	"github.com/heketi/heketi/apps"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+	"net/http"
+)
```

Comment:
Can you please separate the standard library packages from the external packages with an empty newline? You did it that way in app_topology_test.go